src/components/expanded-state/unique-token/NFTBriefTokenInfoRow.tsx

```diff
@@ -2,68 +2,57 @@ import lang from 'i18n-js';
 import React, { useCallback, useEffect, useState } from 'react';
 import { TokenInfoItem } from '../../token-info';
 import { Columns } from '@/design-system';
-import { apiGetUniqueTokenFloorPrice } from '@/handlers/opensea-api';
-import { Network } from '@/helpers';
 import { useAccountSettings } from '@/hooks';
 import { useNavigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
 import { useTheme } from '@/theme';
-import { convertAmountToNativeDisplay } from '@/helpers/utilities';
+import {
+  convertAmountToNativeDisplay,
+  convertRawAmountToDecimalFormat,
+} from '@/helpers/utilities';
 import { ethereumUtils } from '@/utils';
+import { UniqueAsset } from '@/entities';
+import { fetchSimpleHashNFTListing } from '@/resources/nfts/simplehash';
 
 const NONE = 'None';
 
-const getIsFloorPriceSupported = (network: Network) => {
-  switch (network) {
-    case Network.mainnet:
-      return true;
-    default:
-      return false;
-  }
-};
-
 export default function NFTBriefTokenInfoRow({
-  currentPrice,
-  lastPrice,
-  lastSalePaymentToken,
-  network: assetNetwork,
-  urlSuffixForAsset,
+  asset,
 }: {
-  currentPrice?: number | null;
-  lastPrice?: number | null;
-  lastSalePaymentToken?: string | null;
-  network: Network;
-  urlSuffixForAsset: string;
+  asset: UniqueAsset;
 }) {
   const { colors } = useTheme();
 
   const { navigate } = useNavigation();
 
-  const { nativeCurrency, network } = useAccountSettings();
+  const { nativeCurrency } = useAccountSettings();
 
-  const [floorPrice, setFloorPrice] = useState<string | null>(null);
+  const floorPrice =
+    asset?.floorPriceEth !== undefined
+      ? `${asset?.floorPriceEth !== '0' ? asset?.floorPriceEth : '< 0.001'} ETH`
+      : NONE;
 
-  useEffect(() => {
-    const isFloorPriceSupported = getIsFloorPriceSupported(assetNetwork);
+  const [currentPrice, setCurrentPrice] = useState<string | null>();
 
-    const fetchFloorPrice = async () => {
-      if (isFloorPriceSupported) {
-        try {
-          const result = await apiGetUniqueTokenFloorPrice(
-            network,
-            urlSuffixForAsset
-          );
-          setFloorPrice(result);
-        } catch (_) {
-          setFloorPrice(NONE);
-        }
-      } else {
-        setFloorPrice(NONE);
+  useEffect(() => {
```

Comment:
simplehash nft response doesn't provide current price, so we have to fetch it here. it does, however, provide floor price, so we can get rid of that